lib/sessions/session-oauth.js

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
I would like to remove this completely. I think it's better to have this as a separate module, see https://github.com/lpinca/shopify-token.